### DISCUSSION TITLE
Student Benefits Application Pending for 3 Days

### DISCUSSION THREAD

**Discussion by Ivy-js** [community/community#143898] [GitHub Education] [answered]
### Hi everyone,

Hello GitHub Education Team,

I'm reaching out to inquire about my student benefits application. It’s been three days since I submitted my application, and I haven’t received any update yet. I'm Ivy (@ivy-js), and I'm really looking forward to using these benefits to further my learning and projects. Could someone please provide an update or let me know if there are any issues with my application?

Thank you for your time and assistance!

Best regards,
Ivy (@ivy-js)

**Comment by davevad93** [comment, marked as answer]
Hi @Ivy-js ,

GitHub no longer provides manual reviews, when the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Ivy-js was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
